ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/itemTemplates/card.html

```diff
@@ -0,0 +1,20 @@
+<template data-sly-template.layout="${@ hideTime, hideFooter}">
+    <div data-cmp-hook-item-template="includes">
+        <sly data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+             data-sly-call="${clientlib.all @ categories=['core.forms.components.commons.v1.clientlibs.layouts.card']}"></sly>
+    </div>
+    <div class="cmp-portallister-results" data-cmp-hook-item-template="container"></div>
```

Comment:
Shouldn't this be cmp-portallister__results as per BEM if the block is cmp-portallister and element is the results ?

Reply:
I intended the whole `cmp-portallister-results` as a seperate block and `cmp-portallister` as a seperate block. The first one is container (it'll be a table in list view) which holds all the listings.
Second one is a single listing and has it's seperate elements. 

Let's discuss this if the rational isn't clear.

Reply:
Updated to follow one block per component convention